src/foundry/client/core/keybindings.d.ts

```diff
@@ -166,16 +164,14 @@ declare class ClientKeybindings {
   /**
    * Handle Dismiss actions
    * @param context - The context data of the event
-   *                  (unused)
    * @internal
    */
   protected static _onDismiss(context?: KeyboardEventContext): boolean;
 
   /**
    * Open Character sheet for current token or controlled actor
-   * @param event   - (unused)
+   * @param event   -
```

Comment:
Shouldn't we just remove the entire `@param` here?

Reply:
The problem is that the second parameter has some description, and if we remove the first one, it doesn’t match up properly anymore and is displayed incorrectly by editors (the documentation of the second parameter is displayed for the first one).

Reply:
Ewww! I would have thought most editors do that by name instead of index! Fine then, we keep it like this.